includes/fbproduct.php

```diff
@@ -845,7 +944,11 @@ function( $key ) {
 		$matched_attributes = array_filter(
 			$all_attributes,
 			function( $attribute ) use ( $sanitized_keys ) {
-				return in_array( $attribute['key'], $sanitized_keys );
+				// Check if $attribute is an array and has the 'key' index
+				if (is_array($attribute) && isset($attribute['key'])) {
```

Comment:
Formatting, check also through the code

```suggestion
				if ( is_array( $attribute ) && isset( $attribute['key'] ) ) {
```